pkg/operator/sync.go

```diff
@@ -136,17 +138,19 @@ func (optr *Operator) syncBaremetalControllers(config *OperatorConfig) error {
 	}
 	if updated {
 		resourcemerge.SetDeploymentGeneration(&optr.generations, d)
-		return optr.waitForDeploymentRollout(metal3Deployment)
+		return optr.waitForDeploymentRollout(metal3Deployment, deploymentRolloutPollInterval, deploymentRolloutTimeout)
 	}
 
 	return nil
 }
 
-func (optr *Operator) waitForDeploymentRollout(resource *appsv1.Deployment) error {
+func (optr *Operator) waitForDeploymentRollout(resource *appsv1.Deployment, deploymentRolloutPollInterval, deploymentRolloutTimeout time.Duration) error {
```

Comment:
These names don't need to be so long/descriptive while they're in this method, could shorten them + then they don't shadow the package level consts
```suggestion
func (optr *Operator) waitForDeploymentRollout(resource *appsv1.Deployment, pollInterval, timeout time.Duration) error {
```